db/sqlglue.c

```diff
@@ -12488,7 +12516,8 @@ int indexes_expressions_data(struct schema *sc, const char *inbuf, char *outbuf,
         tzname = "America/New_York";
 
     sql = strbuf_new();
-    memset(&mout, 0, sizeof(Mem));
+    mout = (Mem *)malloc(sizeof(Mem));
```

Comment:
Sure. I was just trying to be safe as this is passed to another thread (`dispatch_sql_query`). But this function wont exit until the sql returns. So I think it is ok to put it back to stack.